cpu/esp32/Makefile.include

```diff
@@ -1,16 +1,17 @@
 # ESP32x specific flashing options
-FLASH_CHIP  = $(CPU)
-ifneq (,$(filter esp32,$(CPU)))
+FLASH_CHIP  = $(CPU_FAM)
+ifneq (,$(filter esp32,$(CPU_FAM)))
```

Comment:
```suggestion
ifeq (esp32,$(CPU_FAM))
```

`CPU_FAM` is not a list

Reply:
Yes, I will change it for the moment. However, in future PRs for ESP32-S2 and ESP32-S3 were multiple families will match, we will need the `filter` function.

Reply:
Ah true, if this gets expanded in the future filter is warranted - anyway feel free to squash